### DISCUSSION TITLE
Replace base control for a new UI theme

### DISCUSSION THREAD

**Discussion by elgransan** [oqtane/oqtane.framework#2165] [Q&A]
Hi, here working with this fabulous framework, I'm trying to integrate [Material Design for Boostrap](https://mdbootstrap.com/) It was easy the first part, and looks great (as you can see) 
![image](https://user-images.githubusercontent.com/9949584/166170380-724087fa-649a-4ad5-bf19-bbee512289a5.png)

But some functionality as tabstrips and accordion are not working and I'm not sure how to continue, as I can see MDB replaced all the bootstrap "data-bs-xxxx" tags for "data-mdb-xxxx" tags so all this components are affected:

![image](https://user-images.githubusercontent.com/9949584/166170478-d183c8de-a404-4cd9-bba8-e6906bc4f669.png)

I can correct those that are part of the theme folder, but it is possible to replace from my custom theme project the "Section" and "TabStrip" base controls?

**Comment by sbwalker** [comment]
The Oqtane administrative user interface uses Bootstrap 5 as its foundational CSS framework.
